libs/cdk/src/lib/utils/directives/focusable-list/focusable-list.directive.ts

```diff
@@ -239,15 +375,27 @@ export class FocusableListDirective implements OnChanges, AfterViewInit {
                 tap((event: KeyboardEvent) => {
                     this._keydown$.next({ list, event, activeItemIndex: this._keyManager?.activeItemIndex });
 
+                    // Already handled
                     if (event.defaultPrevented) {
                         return;
                     }
 
+                    // Prevent scrolling and other default actions
+                    // But allow tabbing in/out and F2 to jump into list
+                    if (!KeyUtil.isKeyCode(event, [TAB, F2])) {
```

Comment:
We still need to be able to click buttons via keyboard, enter text into inputs, etc when these are all in focusable lists

Reply:
That's a nice catch, thanks. Fixed.